src/pages/NewPost.tsx

```diff
@@ -35,10 +37,10 @@ export const NewPost = (): JSX.Element => {
 
   return (
     <ThemeProvider theme={defaultTheme}>
```

Comment:
Same issue here, doesn't it make sense to have the ThemeProvider at a higher level? Especially since you're using it in more than one place